website/docs/r/container_group.html.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_group"
+sidebar_current: "docs-azurerm-resource-container-group"
+description: |-
+  Create as an Azure Container Group instance.
+---
+
+# azurerm\_container\_group
+
+Create as an Azure Container Group instance.
+
+## Example Usage (Linux)
+
+```hcl
+resource "azurerm_resource_group" "aci-rg" {
+  name     = "aci-test"
+  location = "west us"
+}
+
+resource "azurerm_container_group" "aci-helloworld" {
+  
+  name = "aci-hw"
+  location = "west us"

```

Comment:
_minor_ can we update this to be `"${azurerm_resource_group.aci-rg.location}"`

Reply:
Done.